colors/jellybeans.vim

```diff
@@ -65,7 +65,7 @@ endif
 
 let colors_name = "jellybeans"
 
-if has("gui_running") || &t_Co == 88 || &t_Co == 256
+if has("gui_running") || (exists('&tgc') && &termguicolors) || &t_Co >= 88
```

Comment:
How about `has('termguicolors')` instead of `exists('&tgc')`? Or use the full `exists('&termguicolors')`.

`&t_Co >= 88` basically bets that future terminal color formats will be compatible with either xterm256 or termguicolors, which seems reasonable to me.